benches/geninput.py

```diff
@@ -138,7 +139,33 @@ def chunks(l, n):
 
             i += ATLAS_WIDTH
 
-        with open(decode_input, 'wb') as fn:
+        with open(decode_input_nv12, 'wb') as fn:
```

Comment:
I think we can avoid code duplication since in nv12 and i420 luma array is the same. You can compute CbCr needed for nv12 and compute in the same original loop also Cb and Cr (they are distinct) needed for i420.

Then you could write the data:
```
        with open(decode_input_nv12, 'wb') as fn:
            fn.write(('P5\n%d %d\n255\n' % (ATLAS_WIDTH, height + height // 2)).encode('utf-8'))
            Y.tofile(fn)
            CbCr.tofile(fn)

        with open(decode_input_i420, 'wb') as fn:
            fn.write(('P5\n%d %d\n255\n' % (ATLAS_WIDTH, height + height // 2)).encode('utf-8'))
            Y.tofile(fn)
            Cb.tofile(fn)
            Cr.tofile(fn)
```